ballerina/types.bal

```diff
@@ -1426,7 +1426,7 @@ public class ProcedureCallResult {
     }
 
     # Updates `executionResult` or `queryResult` field with the succeeding result in the result list. This will also close the current
-    # result when called.
+    # result when called. Once all results are processed, the `close()` method must be called.
```

Comment:
These are changed in https://github.com/ballerina-platform/module-ballerina-sql/pull/609